chain/vm/vm.go

```diff
@@ -254,6 +254,9 @@ func (vm *VM) send(ctx context.Context, msg *types.Message, parent *Runtime,
 		if aerr := rt.chargeGasSafe(rt.Pricelist().OnMethodInvocation(msg.Value, msg.Method)); aerr != nil {
 			return nil, aerrors.Wrap(aerr, "not enough gas for method invocation")
 		}
+
+		// not charging any gas, just logging
+		//nolint:errcheck
```

Comment:
OT: We charge gas _5_ times here (well, charge once, log 4 times), computing the same stack trace each time. This is one of our most expensive operations, IIRC. We should compute a trace _once_.

cc @Kubuxu

Reply:
Yeah, these charges are for timestamp/duration evaluation.
We can probably disable traces for gas charges of 0.